src/webportal/prettier.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // Why include an unnecessary character at the end of every line? Break the habit (automatically)!
+  semi: false,
```

Comment:
Do we want to remove the semicolon? Seems most JS code styles use semicolon to avoid some errors.
refer: [airbnb code style](https://github.com/airbnb/javascript#semicolons)

Reply:
Actually we considered airbnb as a candidate rule, however there is a voice that the airbnb config is too much overcontrol in some places and contains more plugins than actually needed. So we choose standard instead.
Standard uses a semicolon-free style, and it has [the solution to avoid certain errors](https://standardjs.com/rules.html#semicolons). And since we use prettier as a formatter, it could [automatically resolve the issue for us](https://prettier.io/docs/en/rationale.html#semicolons).

Reply:
add semi as discussion